src/components/WalletsPage/index.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import { Link, Switch, Route } from 'react-router-dom';
+import compose from 'recompose/compose';
+import { withStyles } from '@material-ui/core/styles';
+import Tabs from '@material-ui/core/Tabs';
+import Tab from '@material-ui/core/Tab';
+import Divider from '@material-ui/core/Divider';
+import Typography from '@material-ui/core/Typography';
+import VerticalAlignBottomIcon from '@material-ui/icons/VerticalAlignBottom';
+import VerticalAlignTopIcon from '@material-ui/icons/VerticalAlignTop';
+import Hidden from '@material-ui/core/Hidden';
+import Deposit from '../../containers/WalletsPage/Deposit';
+import Withdraw from '../../containers/WalletsPage/Withdraw';
```

Comment:
I'm not sure we need to import Deposit and Withdraw

Reply:
@AntonBurduzha